package/debian/Dockerfile

```diff
@@ -7,7 +7,7 @@ ARG LLVM_VERSION=8
 ENV TZ America/Chicago
 ENV DEBIAN_FRONTEND=noninteractive
 
-RUN if [ "$BASE_IMAGE" = "debian:buster" ]; then echo "Enabling backports..."; echo "deb http://deb.debian.org/debian buster-backports main" > /etc/apt/sources.list.d/buster-backports.list; fi
+# RUN if [ "$BASE_IMAGE" = "debian:bullseye" ]; then echo "Enabling backports..."; echo "deb http://deb.debian.org/debian bullseye-backports main" > /etc/apt/sources.list.d/bullseye-backports.list; fi
```

Comment:
Can we just delete this line instead? We will have it in th eGit history.

Reply:
I want to leave it as a comment because I am anticipating needing to uncomment it sometime prior to Debian 12 being released. It will be easier to revive if we don't have to go digging around in multiple-year-old git history to find it again.